plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/ResultSetModel.java

```diff
@@ -1123,7 +1123,7 @@ public void resetOrdering() {
                     }
                     Object cell1 = getCellValue(new ResultSetCellLocation(binding, row1));
                     Object cell2 = getCellValue(new ResultSetCellLocation(binding, row2));
-                    result = DBUtils.compareDataValues(cell1, cell2);
+                    result = DBUtils.compareDataValues((Object) cell1.toString().toLowerCase(), (Object) cell2.toString().toLowerCase());
```

Comment:
Please do such comparison only for string types. It makes no sense to do it for numbers or binary objects.